Methane_MarkDown.Rmd

```diff
@@ -111,3 +110,206 @@ ggplot(data= results, aes(x= year, y= value, color= units))+
          color = "Units") + facet_wrap(~variable)
```

Comment:
It won't let me leave a comment by line 95 but maybe add a graph title so it's clear what you're showing